src/main/groovy/org/labkey/gradle/task/DeployApp.groovy

```diff
@@ -126,9 +126,20 @@ class DeployApp extends DefaultTask
     // Use this method to preserve file permissions, since ant.copy does not, but this does not preserve last modified times
     private void deployBinariesViaProjectCopy(String osDirectory)
     {
-        project.copy { CopySpec copy ->
-            copy.from project.fileTree("${project.labkey.externalDir}/${osDirectory}").files
-            copy.into "${project.serverDeploy.binDir}"
+        File parentDir = new File("${project.labkey.externalDir}", "${osDirectory}")
```

Comment:
You don't need the "${..}" unless you are constructing a string that needs these variable values incorporated. You can just do:
 File parentDir = new File(project.labkey.externalDir, osDirectory)